server.js

```diff
@@ -1090,6 +1090,36 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Codetally integration.
+camp.route(/^\/codetally\/(.*)\/(.*)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var owner = match[1];  // eg, triggerman722.
+  var repo = match[2];   // eg, colorstrap
+  var format = match[3];
+  var apiUrl = 'http://www.codetally.com/formattedshield/' + owner + '/' + repo;
+  var badgeData = getBadgeData('codetally', data);
+  request(apiUrl, function dealWithData(err, res, buffer) {
+    if (err != null) {
+      badgeData.text[1] = 'inaccessible';
+      sendBadge(format, badgeData);
+      return;
+    }
+    try {
+      var data = JSON.parse(buffer);
+      badgeData.text[1] = " " + data.currency_sign + " " + data.amount + " " + data.multiplier;
```

Comment:
Could you omit the space between the currency sign and the amount? **$2.50** instead of **$ 2.50**.